### PR TITLE
docs: implement chapter "GETTING STARTED"

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2bgyqve SDK 文档编写 —— Getting Started](https://app.clickup.com/t/2bgyqve)

File: docs/source/getting_started/10_minutes_to_graviti.rst [hunk 1]
```diff
@@ -4,3 +4,182 @@
...
+       row_data = {
+           "filename": filename,
+           "box2ds": DataFrame(
+               [
+                   {
+                       "xmin": 1,
+                       "ymin": 1,
+                       "xmax": 4,
+                       "ymax": 5,
+                       "category": "bird",
```
Comment (AChenQ) [inline docs/source/getting_started/10_minutes_to_graviti.rst:65]: category `bird` mismatched with the schame `categories=["boat", "car"]`, and `attribute` also.

**Review by AChenQ — CHANGES_REQUESTED** [review]

File: docs/source/getting_started/10_minutes_to_graviti.rst [hunk 2]
```diff
@@ -4,3 +4,182 @@
...
+
+This is a simple introductory tutorial for beginners.
+
+**************
+ Registration
+**************
+
+Before using Graviti SDK, please finish the following registration steps:
+
+- Please visit `Graviti AI Service(GAS)`_ to sign up.
```
Comment (linjiX) [inline docs/source/getting_started/10_minutes_to_graviti.rst:16]: we do not use the word `GAS` anymore

File: docs/source/getting_started/installation.rst [hunk 3]
```diff
@@ -4,3 +4,36 @@
 ##############
  Installation
 ##############
+
+Graviti can be installed with pip or from source. 
+
+**************************
+ Supported Python Version
```
Comment (linjiX) [inline docs/source/getting_started/installation.rst:11]: do we need a section for python version? will it too heavy?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: docs/source/getting_started/10_minutes_to_graviti.rst [hunk 4]
```diff
@@ -4,3 +4,182 @@
...
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.create("Graviti-dataset-demo")
+   draft = dataset.drafts.create("Draft-1")
+
+   ob = pt.build_openbytes("main")
+   box2ds = ob.label.Box2D(
+       categories=["boat", "car"],
+       attributes={"difficult": pt.boolean()},
+   )
+   voc2012 = pt.record(
```
Comment (Lee-000) [inline docs/source/getting_started/10_minutes_to_graviti.rst:46]: `voc2012` or just `schema`?

File: docs/source/getting_started/10_minutes_to_graviti.rst [hunk 5]
```diff
@@ -4,3 +4,183 @@
...
+**************
+ Registration
+**************
+
+Before using Graviti SDK, please finish the following registration steps:
+
+- Please visit `Graviti`_ to sign up.
+- Please visit `Graviti Developer Tools`_ to get an AccessKey.
+
+.. _Graviti: https://gas.graviti.com/tensorbay/
```
Comment (linjiX) [inline docs/source/getting_started/10_minutes_to_graviti.rst:19]: why the first url has a trailing slash, but second does not?

Comment (graczhual) [reply docs/source/getting_started/10_minutes_to_graviti.rst:19]: I'm not sure what the reason is, but this is the url of current website

Comment (linjiX) [reply docs/source/getting_started/10_minutes_to_graviti.rst:19]: ref:
https://searchfacts.com/url-trailing-slash/

**Review by linjiX — APPROVED** [review]

**Review by AChenQ — APPROVED** [review]

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**